src/services/refactors/extractMethod.ts

```diff
@@ -293,14 +293,13 @@ namespace ts.refactor.extractMethod {
             }
 
             let errors: Diagnostic[];
-            let permittedJumps = PermittedJumps.Return;
             let seenLabels: Array<__String>;
 
-            visit(nodeToCheck);
+            visit(nodeToCheck, PermittedJumps.Return);
 
             return errors;
 
-            function visit(node: Node) {
+            function visit(node: Node, permittedJumps: PermittedJumps) {

```

Comment:
this is rather a style comment. we have used passing functions as arguments to forEach, forEachChild, map, etc..

Reply:
Sorry, I'm not sure I understand this comment.  Were you saying that it was more convention before, when we just captured a variable and reset it after each recursive call?

Reply:
Fair enough.  I'll just revert 57906fe90e8efd2fb285fcb67f018c0438ba06dd.